README.md

```diff
@@ -25,15 +26,16 @@ The DocSearch scraper will use a configuration file specifying:
 Once you run the DocSearch scraper on a specific configuration, it will:
  - crawl all the URLs you specified (from the *start_urls* or the *sitemap*)
  - follow all the hyperlinks mentioned in the page, and continue the crawling there
- - stop the crawling as soon as you've reached a URL that is not specified in your configuration or affiliated to a start url
+ - stop the crawling as soon as you've reached a URL that is not specified in your
+ configuration or affiliated with a start url
  - extract the content of every single crawled page following the logic you defined using the CSS selectors
  - push the resulting records to the Algolia index you configured
 
 ## Configuration format
 
 A configuration file looks like:
 
-```json
+```javascript
```

Comment:
this is JSON, not JS, the errors are expected IMO

Reply:
Yes for sure we are aware, we want to see the different display of it on Github in order to make it better. It was a try.